zuul.d/projects.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- project:
```

Comment:
isn't project's name required here?
`    name: openstack-k8s-operators/ci-bootstrap`

Reply:
Yes, it is required. Updated it in the new patchset.

Reply:
`name: name:` typo